SI/Sem.02/Pract.02/task_9_p02.cpp

```diff
@@ -0,0 +1,52 @@
+
+
+#include <iostream>
+
+int main()
+{
+    int a = 0;
+    int b = 0;
+    int c = 0;
+    std::cout << "a= ";
+    std::cin >> a;
+    std::cout << "b= ";
+    std::cin >> b;
+    std::cout << "c= ";
+    std::cin >> c;
+    if (a < b && a<c)
```

Comment:
Think of a better way to do this task
Hint: swap the numbers between them if some condition is true